lib/lte_link_control/lte_lc.c

```diff
@@ -167,13 +175,30 @@ static const enum lte_lc_system_mode sys_mode_fallback =
 
 static enum lte_lc_system_mode sys_mode_current = LTE_LC_SYSTEM_MODE_NONE;
 
-/* Parameters to be passed when using AT%XSYSTEMMMODE=<params> */
+/* Parameters to be passed using AT%XSYSTEMMMODE=<params>,<preference> */
 static const char *const system_mode_params[] = {
-	[LTE_LC_SYSTEM_MODE_LTEM]	= "1,0,0,0",
-	[LTE_LC_SYSTEM_MODE_NBIOT]	= "0,1,0,0",
-	[LTE_LC_SYSTEM_MODE_GPS]	= "0,0,1,0",
-	[LTE_LC_SYSTEM_MODE_LTEM_GPS]	= "1,0,1,0",
-	[LTE_LC_SYSTEM_MODE_NBIOT_GPS]	= "0,1,1,0",
+	[LTE_LC_SYSTEM_MODE_NONE]		= "0,0,0",
+	[LTE_LC_SYSTEM_MODE_LTEM]		= "1,0,0",
+	[LTE_LC_SYSTEM_MODE_NBIOT]		= "0,1,0",
+	[LTE_LC_SYSTEM_MODE_GPS]		= "0,0,1",
+	[LTE_LC_SYSTEM_MODE_LTEM_GPS]		= "1,0,1",
+	[LTE_LC_SYSTEM_MODE_NBIOT_GPS]		= "0,1,1",
+	[LTE_LC_SYSTEM_MODE_LTEM_NBIOT]		= "1,1,0",
+	[LTE_LC_SYSTEM_MODE_LTEM_NBIOT_GPS]	= "1,1,1",
+};
+
+/* LTE preference to be passed using AT%XSYSTEMMMODE=<params>,<preference> */
+static const char system_mode_preference[] = {
```

Comment:
The alignment in the initialization list seems to be broken.

Reply:
Fixed